src/Compilers/CSharp/Test/Symbol/Symbols/PartialPropertiesTests.cs

```diff
@@ -1648,6 +1685,92 @@ partial class C<T>
             Assert.Equal(NullableAnnotation.Oblivious, p3.TypeWithAnnotations.NullableAnnotation);
         }
 
+        [Fact]
+        public void NullableDifference_Analysis()
+        {
+            // Nullable annotations from the *definition* part parameters are used when analyzing implementations.
```

Comment:
>*definition* part parameters are used when analyzing implementations. [](http://example.com/codeflow?start=45&length=69)

Is this statement correct? If we used the definition part parameters for analyzing the implementation, why are reporting warnings `// 3` and `// 4`?

Reply:
You're right, I'll adjust the comment and the spec PR. Don't know how I got mixed up on this.